compiler/parser/parser.peg

```diff
@@ -876,7 +876,7 @@ SQLFrom
 
 SQLAlias
   = _ AS _ id:Lval { RETURN(id) }
-  / _ id:Lval { RETURN(id) }
+  / !(_ SQLTokenSentinels)_ id:Lval { RETURN(id) }
```

Comment:
```suggestion
  /  _ !SQLTokenSentinels id:Lval { RETURN(id) }
```

Reply:
Unless I am missing something you need to ensure the space after the SQLTokenSentinels exists otherwise you can't use ids that start with a token like `fromt`.

Reply:
Oh. Tests passed with this change so I figured that meant it was a winner.

Reply:
I should probably write a test where this fails!

Reply:
I just tested this out and this one needs to remain as written, but I'll add test for this.